perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -70,6 +70,10 @@
 # by EXECUTE_COMMAND.
 WAIT_FOR_COMMAND = 'wait_for_command.py'
 
+flags.DEFINE_bool('setup_host_firewall', False,
```

Comment:
Can you called this setup_guest_firewall.  Host generally means the server running the virtual machine so I'd like to reduce confusion.